docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -138,45 +142,149 @@ You can initialize a selector with any of these objects.
     });
     ```
 
-### Initializers that Return Multiple Nodes
+### Check Selector Matching Set Length
+
+Functions and CSS selector strings which initialize a selector can have one matching DOM element on the page, several or nothing.
```

Comment:
Functions and CSS selector strings that initialize a selector may return a single matching DOM element on the page, multiple elements or nothing.